src/main/java/com/minecolonies/util/BlockUtils.java

```diff
@@ -41,6 +42,29 @@
 
       );
 
+
+    /**
+     * Updates the rotation of the structure depending on the input.
+     *
+     * @param rotation the rotation to be set.
+     * @return returns the Rotation object.
+     */
+    public static Rotation getRotation(final int rotation)
+    {
+        switch (rotation)
+        {
+            case 1:
+                return Rotation.CLOCKWISE_90;
+            case 2:
+                return Rotation.CLOCKWISE_180;
+            case 3:
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 3 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)